core/server/master/src/main/java/alluxio/master/metastore/rocks/RocksBlockMetaStore.java

```diff
@@ -62,7 +69,9 @@ public class RocksBlockMetaStore implements BlockMetaStore {
   // This is a field instead of a constant because it depends on the call to RocksDB.loadLibrary().
   private final WriteOptions mDisableWAL;
   private final ReadOptions mIteratorOption;
-  private final ColumnFamilyOptions mColumnFamilyOptions;
+  private final ReadOptions mReadPrefixSameAsStart;
+
+  private final List<RocksObject> mToClose = new ArrayList<>();
```

Comment:
Would `Closer.register()` work better?

Reply:
The rocks objects are AutoCloseable, but the Closer that I found only takes Closeable.